core/vdbe/execute.rs

```diff
@@ -8664,7 +8664,8 @@ impl Value {
         .max(1) as usize;
 
         let mut blob: Vec<u8> = vec![0; length];
-        getrandom::getrandom(&mut blob).expect("Failed to generate random blob");
+        rand::thread_rng().fill_bytes(&mut blob);
```

Comment:
Not a bug you added, but this is not deterministic under simulation. We need to add `getrandom` to I/O to fix this.